lib/recurly/account_acquisition.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+/**
+ * Class Recurly_AccountAcquisition
+ * @property Recurly_Stub $account The associated Recurly_Account for this acquisition object.
```

Comment:
We refer to this object as "acquisition object", "account acquisition", "account acquisition info" can we just pick one and use it consistently?